Sources/Sentry/SentryProfiler.mm

```diff
@@ -585,12 +588,15 @@ - (void)captureEnvelope
                           : (unsigned long long)(
                               [transaction.timestamp timeIntervalSinceDate:_startDate] * 1e9)];
         [transactionsInfo addObject:@{
-            @"environment" : _hub.scope.environmentString ?: _hub.getClient.options.environment ?: kSentryDefaultEnvironment,
             @"id" : transaction.eventId.sentryIdString,
             @"trace_id" : transaction.trace.context.traceId.sentryIdString,
             @"name" : transaction.transaction,
             @"relative_start_ns" : relativeStart,
-            @"relative_end_ns" : relativeEnd
+            @"relative_end_ns" : relativeEnd,
```

Comment:
Do we have a TODO somewhere to make sure the `relative_end_ns` is correct?

Reply:
I just filed an issue here for this, thanks for reminding me! https://github.com/getsentry/sentry-cocoa/issues/2352